internal/hmyapi/apiv1/blockchain.go

```diff
@@ -217,6 +214,20 @@ func (s *PublicBlockChainAPI) GetValidators(ctx context.Context, epoch int64) (m
 	return result, nil
 }
 
+// GetValidatorKeys returns list of bls public keys in the committee for a particular epoch.
+func (s *PublicBlockChainAPI) GetValidatorKeys(ctx context.Context, epoch int64) ([]string, error) {
```

Comment:
can you implement this function in the backend and api1 and api2 can directly call and return it?